lib/core/src/Cardano/Wallet/Primitive/Types/UTxO.hs

```diff
@@ -151,6 +153,9 @@ balance =
 difference :: UTxO -> UTxO -> UTxO
 difference a b = a `excluding` Map.keysSet (unUTxO b)
 
+intersection :: UTxO -> UTxO -> UTxO
+intersection (UTxO a) (UTxO b) = UTxO (a `Map.intersection` b)
```

Comment:
```suggestion
intersection (UTxO a) (UTxO b) = UTxO (Map.intersection a b)
```

Reply:
@Unisay I agree that it's nice to avoid mixing infix- and prefix-style here.

I've converted these to use prefix style in f36e0a45fea20f3e64b09ba26cdf76b8a66d4fd2.